springfield/firefox/templates/firefox/landing/windows.html

```diff
@@ -0,0 +1,313 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ #}
+
+{% from "macros.html" import google_play_button, apple_app_store_button, fxa_email_form with context %}
+
+{% set params = '?utm_source=www.firefox.com&utm_medium=referral&utm_campaign=windows-landing' %}
+{% set ios_url = app_store_url('firefox', 'windows-landing') %}
```

Comment:
Can we include `windows-landing` for android too?

`{% set android_url = play_store_url('firefox', 'windows-landing') %}`